app/views/layout.html

```diff
@@ -18,4 +18,5 @@
 
 {% block body_end %}
   {% include "includes/scripts.html" %}
+  <!-- GOV.UK Prototype kit {{releaseVersion}} -->
```

Comment:
Does that add any value for us? It makes the comment more noisy and less readable IMHO.